pippo-session-parent/pippo-session-spymemcached/src/main/java/ro/pippo/session/spymemcached/SpymemcachedSessionDataStorage.java

```diff
@@ -69,7 +69,6 @@ public SessionData get(String sessionId) {
         if (obj == null) {
             return null;
         }
-        this.sessions.touch(sessionId, idleTime);
         SessionData sessionData = (SessionData) obj;
```

Comment:
Just a curiosity. Why `return (SessionData) this.sessions.get(sessionId);` is not OK for you?
The actual code is:
```java
Object obj = this.sessions.get(sessionId);
return obj == null ? null : (SessionData) obj;
```
but in my opinion `obj == null ? null : (SessionData) obj;` is not needed, it's redundant.
It's only a curiosity, nothing wrong.